benchmarks/src/jmh/java/io/grpc/benchmarks/netty/AbstractBenchmark.java

```diff
@@ -369,8 +381,15 @@ public void onComplete() {
 
                       @Override
                       public void onReady() {
-                        while (call.isReady()) {
-                          call.sendMessage(response.slice());
+                        // Cork while the transport declares that it can receive more messages
+                        // so that flushes are deferred until the framer buffer is full.
+                        call.cork();
```

Comment:
This code makes me feel like it would possibly be better served with an AutoCloseable. Did you discover some reasons to use the current approach, or did you just "go with something" (since there would be bikeshedding in any case).